packages/components/modal/src/ModalConfirm/README.mdx

```diff
@@ -5,48 +5,65 @@ status: 'stable'
 section: 'modalComponents'
 slug: /components/modal-confirm/
 github: 'https://github.com/contentful/forma-36/tree/master/packages/components/modal/src'
-storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-modal-modalconfirm--basic'
 typescript: ./ModalConfirm.tsx
 ---
 
-import { Props } from '@contentful/f36-docs-utils';
+`ModalConfirm` is a convenience wrapper on top of [Modal]. It is intended to use for showing confirmation dialogs.
 
-`ModalConfirm` is a convenience wrapper on top of `Modal`. It is intended to use for showing confirmation dialogs.
-
-### Import
+## Import
 
 ```js static=true
 import { ModalConfirm } from '@contentful/f36-components';
-// or
-import { ModalConfirm } from '@contentful/f36-modal';
```

Comment:
Recently we discovered that there are apps where tree shaking is not working, like UI extensions (if I remember the name right). So we decided to keep the examples with separate imports.

Reply:
ah, do we have two options in all examples now?

Reply:
yes, we did a follow up PR yesterday: https://github.com/contentful/forma-36/pull/1727